kafka-rest/src/test/java/io/confluent/kafkarest/integration/ResourceDiscoverabilityTest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest.integration;
+
+import static org.junit.Assert.assertEquals;
+
+import io.confluent.kafkarest.KafkaRestConfig;
+import io.confluent.kafkarest.extension.RestResourceExtension;
+import java.util.Collections;
+import java.util.Properties;
+import javax.annotation.Priority;
+import javax.ws.rs.Priorities;
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
+import javax.ws.rs.core.Configurable;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import org.junit.Test;
+
+public class ResourceDiscoverabilityTest extends ClusterTestHarness {
```

Comment:
ResourceBlocklistFeatureTest would make this test more discoverable (pun intended).

Reply:
Do you mean merging the two tests?

I was thinking about making this test a part of `ResourceBlocklistFeatureTest`, by tweaking the fake auth filter to 401 a specific endpoint only (that's not used in the other tests).
In the end I decided against that, as it makes the single resulting test harder to understand as the fake auth filter applies to just one of the test methods.
Let me know if you still prefer having me merge the two tests - if you don't, I'll keep these as they are.

Reply:
Ack.